tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Utility/Constants/ServiceConstants.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Legal.MatterCenter.Utility
     public class ServiceConstants
     {
 
+        public const int Matter_ONE_NOTE_LENGTH = 120;

```

Comment:
Please add comment explaining why Matter_ONE_NOTE_LENGTH set to 120 in your next checkin...